docs/config-contrib.md

```diff
@@ -0,0 +1,100 @@
+# Config Contributions
+
+## Community Presets
+
+These are a collection of presets from the community. 
+
+!!! warning
+    These presets are built from feedback from the community, they are not routinely tested. Use at your own risk.
+
+### Filter SSO Resources
+
+This is a preset to filter out AWS SSO resources.
+
+```yaml
+presets:
+  sso:
+    filters:
+      IAMSAMLProvider:
+        - type: "regex"
+          value: "AWSSSO_.*_DO_NOT_DELETE"
+      IAMRole:
+        - type: "glob"
+          value: "AWSReservedSSO_*"
+      IAMRolePolicyAttachment:
+        - type: "glob"
+          value: "AWSReservedSSO_*"
```

Comment:
This could also have:

IAMRolePolicy:
  - property: "role:RoleName"
    type: glob
    value: "AWSReservedSSO_*"